shared/chat/manage-channels/edit-channel.js

```diff
@@ -30,14 +40,20 @@ const EditChannelBare = (props: Props & TextState) => (
     <Text type="BodySmallSemibold" style={{color: globalColors.darkBlue, marginTop: globalMargins.xtiny}}>
       {props.teamname}
     </Text>
-    <Text type="Header" style={{marginBottom: globalMargins.tiny, marginTop: globalMargins.tiny}}>
-      Edit #{props.channelName}
-    </Text>
+    {props.waitingForGetInfo ? (
```

Comment:
Hm, I'm not seeing these spinners. I think `waitingForGetInfo` is missing in `mergeProps`

Reply:
D'oh! Added types and fixed